bundles/best_practices.rst

```diff
@@ -463,6 +463,17 @@ API is being used. The following code, would work for *all* users::
         }
     }
 
+Resources
+---------
+
+If the bundle references any resources (config files, templates, translation
```

Comment:
I've reworded this. I truly dislike that the logical path of templates is different than for the rest of resources. I understand that "old Symfony" used bundles everywhere and ``@App/Default/index.html.twig`` was more convenient than ``@AppBundle/Resources/views/Default/index.html.twig``.

However, for Symfony 4 / Flex, which removes the concept of bundles for your own code, I think we should remove this difference and use the same syntax for everything.